en-US/Language.xml

```diff
@@ -1017,6 +1017,8 @@
 			Do not use "master" when it is paired with "slave".
       Such use diminishes the horror of the dehumanizing practice of slavery.
       Use of "master" is acceptable in other contexts, such as to refer to mastery of a skill.
+			 The master and slave mount propagation terminology that is used in Linux is problematic and divisive, and needs to be changed. 
+			 When naming the default branch name for a GitHub repository, use "main" instead of "master.
```

Comment:
```suggestion
			 When naming the default branch name for a GitHub repository, use "main" instead of "master".
			 Use of "master" is acceptable in other contexts, such as to refer to mastery of a skill.
```